airbyte-workers/build.gradle

```diff
@@ -8,6 +8,15 @@ plugins {
 
 application {
     mainClass = 'io.airbyte.workers.process.KubeProcessBuilderFactoryPOC'
+    applicationDefaultJvmArgs = ['-Xmx500m',
```

Comment:
For remote debugging. Will remove before merging into branch. Leaving this here in case someone wants to take a look at what I saw.

port forwarding the port to localhost 6000 and attaching visual vm to the 6000 will confirm okhttp is the thread leak.